src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherPayeeServiceImpl.java

```diff
@@ -182,4 +182,12 @@ protected String getPayeeEntityId(String payeeIdNumber) {
 
         return entityId;
     }
+
+    @Override
+    public String getPayeeTypeCodeForVendorType(String vendorTypeCode) {
+        if (StringUtils.isBlank(vendorTypeCode)) {
+            return null;
```

Comment:
Should this return null or should this be considered an exception? Should there be a unit test for this case when vendorTypeCode is blank?

Reply:
This blank-value check was just meant as a safeguard against passing a null search value to a Map instance controlled by base code, in case KualiCo ends up changing the Map to a type that doesn't allow null inputs. I'll work on adding some unit test methods to test out various inputs to this method.